packages/gatsby-source-contentful/src/__tests__/__snapshots__/extend-node-type.js.snap

```diff
@@ -70,16 +70,16 @@ Object {
 
 exports[`contentful extend node type resolveResponsiveSizes generates responsive sizes data for images using all options 1`] = `
 Object {
-  "aspectRatio": 0.75,
+  "aspectRatio": 1.1278195488721805,
```

Comment:
Looks like something broke this?

Reply:
This is the outcome of the bugfix for this result `w=450&h=399` https://github.com/gatsbyjs/gatsby/pull/4207/commits/b0c658cf0d9947aff0699547b7675d66cb774515

It returned the original aspect ratio while the resulting one should be returned

Reply:
ahhh :-) cool!